websites_list/news_websites.csv

```diff
@@ -1988,3 +1988,13 @@
 1986,New Indian Express,newindianexpress.com,Right Leaning
 1987,Dainik Bhaskar,bhaskar.com,Pro-NDA
 1988,Dainik Jagran,jagran.com,Pro-NDA
+1989,moneycontrol,moneycontrol.com,Ecnonomic news. Owned by TV18
+1990,Bleacher Report,bleacherreport.com
+1991,Eenadu,eenadu.net,Telugu News
+1992,Great Andhra,greatandhra.com,Regional news on Andhra Pradesh
+1993,Dinamalar,dinamalar.com,National Tamil Daily
+1994,Web Dunia,hindi.webdunia.com,Indpendent News Website focused on news in Hindi
+1995,One India,oneindia.com,Leaning right. Owned by BG Mahesh
+1996,Manorama Online,manoramaonline.com,Malyalam newspaper. Generally reliable
+1997,Rediff News,rediff.com/news,Founded by Ajit Balakrishnan
+
```

Comment:
2000,Crikey inq, https://www.crikey.com.au/, On politics, media, business, the environment and life

Reply:
2001,India Today, https://www.indiatoday.in/india, latest India news & top news on India Today.

Reply:
I made more changes https://github.com/AOSSIE-Org/Social-Street-Smart/compare/master...taylorylee:patch-1